docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,206 @@
+- [GitHub Actions](https://docs.github.com/en/actions) and
+  [continuous integration](https://docs.github.com/en/actions/automating-builds-and-tests/about-continuous-integration)
+  in general;
+- [GitHub Actions Reusable Workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows).
+
+The following tools and concepts are used in `ITKRemoteModuleBuildTestPackageAction`
+and may be helpful or required for contributing:
+
+- [Bash](https://www.gnu.org/software/bash/) and
+  [PowerShell](https://learn.microsoft.com/en-us/powershell/) scripting;
+- [CMake](https://cmake.org/) and [Ninja](https://ninja-build.org/) build tools;
+- [CTest](https://cmake.org/cmake/help/latest/manual/ctest.1.html) and 
+  [CDash](https://cmake.org/cmake/help/book/mastering-cmake/chapter/CDash.html)
+  testing and reporting tools;
+- [Python](https://www.python.org/) scripting;
+- [Python wheel](https://wheel.readthedocs.io/en/stable/) packaging;
+
+## Creating or Modifying a Workflow
+
+GitHub Actions requires that workflows are placed in the [`.github/workflows`](../.github/workflows) directory.
+This is where you will find existing workflows and possibly add new workflows.
+
+To contribute to `ITKRemoteModuleBuildTestPackageAction` you will need to
+first [fork](https://docs.github.com/en/github-ae@latest/get-started/quickstart/fork-a-repo)
+the repository to your user account on GitHub and then
+[clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+the fork to your local machine.
+
+### Creating a Workflow
+
+`ITKRemoteModuleBuildTestPackageAction` is a central repository for GitHub Actions
+reusable workflows for ITK external modules. Before creating a workflow, ask yourself
+the following questions:
+1. Does my workflow address a problem for ITK external modules that is not
+   already solved?
+1. Does my workflow provide functionality that will generalize to various ITK external modules?
+1. Does my workflow provide functionality related to continuous integration such as
+   building, packaging, and/or testing steps?
+
+If the answer to all of the above is "yes", then continue on with your new workflow!
+
+Reference
+[GitHub Actions Reusable Workflow documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow)
+to get started. You will need to:
+1. Create a workflow in the `.github/workflows` folder;
+1. Define workflow inputs;
+1. Define how jobs will be launched;
+1. Define steps that a given job will take.
+
+### Modifying a Workflow
+
+Existing workflows will need to be maintained and updated over time as tools
+and technology progress. Workflow modifications will typically be accepted
+only if:
+1. The modification addresses an existing, documented issue in the reusable
+   workflow that prevents the workflow from operating as intended; or
+2. The modification adds a minor feature update that is deemed suitable
```

Comment:
Can use 1. here as well: Markdown will take care of numbering correctly.

Reply:
Included in squash update 👍